core/src/main/scala/cats/arrow/Arrow.scala

```diff
@@ -69,3 +69,8 @@ import simulacrum.typeclass
   def merge[A, B, C](f: F[A, B], g: F[A, C]): F[A, (B, C)] =
     andThen(lift((x: A) => (x, x)), split(f, g))
 }
+
+object Arrow {
+  implicit val partialFunction: Arrow[PartialFunction] =
+    cats.instances.partialFunction.catsStdInstancesForArrowHierarchy
```

Comment:
keeping up with this new practice of having StdLib's instances in the companion object itself

Reply:
It would be probably enough to have the `ArrowChoice[PartialFunction]` and `CommutativeArrow[PartialFunction]` instances in the `Compose` companion object and the `Strong[PartialFunction]` instance in `Profunctor`'s. Something like
```scala
implicit val catsInstancesForPartialFunction: ArrowChoice[PartialFunction] with CommutativeArrow[PartialFunction] =
    cats.instances.partialFunction.catsStdInstancesForArrowHierarchy
```
and
```scala
implicit val catsStrongForPartialFunction: Strong[PartialFunction] =
    cats.instances.partialFunction.catsStdInstancesForArrowHierarchy
```

Reply:
I see thank you @barambani, had to revisit implicit resolution rules again :)

Reply:
It might be that you already know this but, in case you don't, this approach is brilliantly explained [here](https://meta.plasm.us/posts/2019/09/30/implicit-scope-and-cats/)